sspinn/nn_translator.py

```diff
@@ -128,3 +131,28 @@ def nn_translator(filename, train=False):
     output = (el + spectra, connectivity_matrix)
 
     return output
+
+
+def _multiprocess_preload_data(nproc=28, data_dir='data/data'):
+
+    dir_list = os.listdir(data_dir)
+    file_list = []
+    for f in dir_list:
+        if '.txt' in f:
+            file_list.append(data_dir + '/' + f)
+
+    pool = mltp.Pool(processes=nproc)
+    p = pool.map(nn_translator, file_list)
```

Comment:
Why not just combine these 2 lines? Or does that throw an error?

Reply:
AFAIK there is no problem with combining those lines. However, stylistically, I think it's a good idea to do one thing per line for readability (instantiate a pool of workers, then send those workers to do a particular process). It probably has a very negligible effect on performance.